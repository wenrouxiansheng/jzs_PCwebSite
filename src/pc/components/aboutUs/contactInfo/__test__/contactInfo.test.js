import React from "react";
import renderer from "react-test-renderer";
import ContactInfo from "../index";

describe("PC:<ContactInfo />", () => {
  it("render", () => {
    const wrapper = renderer.create(<ContactInfo />);
    let tree = wrapper.toJSON();

    //渲染测试
    expect(tree).toMatchSnapshot();
  });
});
