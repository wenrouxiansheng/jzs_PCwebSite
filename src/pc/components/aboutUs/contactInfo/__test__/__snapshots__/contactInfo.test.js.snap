// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PC:<ContactInfo /> render 1`] = `
<div
  className="contactInfo"
  style={Object {}}
>
  <div
    className="suspension"
  >
    <img
      alt=""
      src="test-file-stub"
    />
    <div>
      <h1>
        全国总部
      </h1>
      <p>
        北京市通州区新华西街60号万达广场A座30层
      </p>
    </div>
  </div>
  <div
    className="info"
  >
    <h1>
      联系我们
    </h1>
    <h3>
      Contact Us
    </h3>
    <div
      className="phone"
    >
      <div>
        <span>
          咨询电话：
        </span>
        <p>
          <span>
            400-900-8898
          </span>
        </p>
      </div>
      <div>
        <span>
          商务合作：
        </span>
        <p>
          <span>
            010-5956-2885
          </span>
        </p>
      </div>
      <div>
        <span>
          人才招聘：
        </span>
        <p>
          <span>
            010-5956-2862
          </span>
          <span>
            010-5956-2861
          </span>
        </p>
      </div>
    </div>
  </div>
  <div
    className="info"
  >
    <h1>
      官方邮箱
    </h1>
    <h3>
      The official E-mail
    </h3>
    <div
      className="phone"
    >
      <div>
        juzishuyishu2011@163.com
      </div>
    </div>
  </div>
  <div
    className="info"
  >
    <h1>
      微信有约
    </h1>
    <h3>
      WeChat about
    </h3>
    <div
      className="QR"
    >
      <div>
        <img
          alt=""
          src="test-file-stub"
        />
        <p>
          订阅号：juzishujiaoyu
        </p>
      </div>
      <div>
        <img
          alt=""
          src="test-file-stub"
        />
        <p>
          服务号：juzishuyishujiaoyu
        </p>
      </div>
    </div>
  </div>
  <div
    className="info"
  >
    <h1>
      官方微博
    </h1>
    <h3>
      Official micro-blog
    </h3>
    <div
      className="QR"
    >
      <div>
        <img
          alt=""
          src="test-file-stub"
        />
        <p>
          官方微博：桔子树艺术教育培训
        </p>
      </div>
    </div>
  </div>
</div>
`;
